src/views/UserLocation.vue

```diff
@@ -93,6 +146,42 @@
             // Method to update map with provided address
             updateMap(address) {
                 this.$refs.mapView.updateMap(address);
+            },
+            // Method to save the location
+            saveLocation() {
+                if (this.latitude !== null && this.longitude !== null) {
```

Comment:
There is some unexpected behavior due to the latitude and longitude. If you don't get an address with geolocation, these remain null so no value is ever saved. Once you get your location, you can save locations, but the latidute and longitude are not updated to the typed in address, the are still for the geolocation. It'll be fine for the end of sprint 2, but should probably be fixed